### PR TITLE
look what it is

### PR DESCRIPTION
**Pull request by NyaightHazard**
* New song option: InputType (#4)
You can choose Keyboard or DancePad, it effects GlobalOffset and Result!
Also, 2 ways of auto-select ("card" or "connect") are available!

* Daily Random Background (part of #5)
And fix not working Random Background

* some fix and improve included
You can choice No friends (#14)
Use alt image for gif when FailOverlay not displaied (#19)
No Decents or Wayoffs effects label color (in the past I was mistaken, sorry) 
Sync to original SimplyLove-NotITG (text on "juice", sort of options, and comment-out - #7)
~~Remove one header joke because of deleted japanese font~~ (Reconvet due to f09ce36)
Add one transition joke (do you still remember...?)

(I accidentally merged private branch and undo'd this, so PLZ dont mind about some commits)

### CODE REVIEW

File: Scripts/Headers.lua [hunk 1]
```diff
@@ -68,7 +68,6 @@ ScreenSelectMusicHeaderTexts = {
   '??????????????? okay',
   'cool! your point?',
   'https://streamable.com/noooo',
-  'conflict歌います。ズォールヒ～～↑wwwwヴィヤーンタースwwwwwワース フェスツwwwwwwwルオルwwwwwプローイユクwwwwwwwダルフェ スォーイヴォーwwwwwスウェン ネwwwwヤットゥ ヴ ヒェンヴガｒジョｊゴアｊガオガオッガwwwじゃｇｊｊ',
```
Comment (Jecket22) [inline Scripts/Headers.lua:71]: this makes me want to add back the jp font

**Review by Jecket22 — APPROVED** [review]
alright, looks good to me

File: Scripts/Headers.lua (hunk 1 above)
Comment (NyaightHazard) [reply Scripts/Headers.lua]: Now reconvet due to f09ce36

**Comment by NyaightHazard** [comment]
Yeah I don't want to violate the commits

**Pull request closed without merging**